database-commons/src/main/java/io/cdap/plugin/db/batch/sink/AbstractDBSink.java

```diff
@@ -364,10 +381,14 @@ protected String getEscapedTableName() {
       return tableName;
     }
 
+    protected String getSchemaName() {
```

Comment:
Can we name it is a `getDBSchemaName()` to avoid confusion with input schema?

Reply:
this is done